lib/maze/client/appium/bb_devices.rb

```diff
@@ -55,6 +55,48 @@ def get_available_device(device_or_group_name)
             end
           end
 
+          def list_device_groups(access_key)
+            api_client = BitBarApiClient.new(access_key)
+            device_groups = api_client.get_device_group_list
+            unless device_groups['data']
+              puts 'There are no device groups available for the given user access key'
+              exit 1
+            end
+            puts 'BitBar device groups available:'
+            device_groups['data'].each do |group|
+              puts '------------------------------'
+              puts "Group name   : #{group['displayName']}"
+              puts "Group ID     : #{group['id']}"
+              puts "OS           : #{group['osType']}"
```

Comment:
I think we can afford to lose both of these - I don't think we'll need the id for anything (now that the api client has been enhanced) and device group names are likely to include the OS name anyway (?).